test/test_daq_mqtt.py

```
@@ -42,6 +42,40 @@ def test_mqtt_to_influxdb_json_single(machinery, create_influxdb, reset_influxdb
     yield record
 
 
+@pytest_twisted.inlineCallbacks
+@pytest.mark.mqtt
+def test_mqtt_to_influxdb_json_bulk(machinery, create_influxdb, reset_influxdb):
```

Ruff (0.15.0)

[warning] Unused function argument: `machinery`

(ARG001)